etc/docker/Dockerfile

```diff
@@ -65,6 +66,11 @@ COPY --from=builder /usr/ldproxy /usr/bin
 COPY --from=builder /usr/espup /usr/bin
 COPY --from=builder /usr/sccache /usr/bin
 
+RUN case "${TARGETPLATFORM}" in \
```

Comment:
Why this way rather than with `cargo install`?

Reply:
actually we don't need it anymore since we use build.rs to generate bindings. But the idea was that I didn't want to include it in arm version of the docker image